samples/x-iac-pipeline/README.md

```diff
@@ -0,0 +1,182 @@
+
+<p align="center">
+  <img src="./sample-architecture.png?raw=true" alt="Apigee X Shared VPC Multi Region Sample Architecture">
+</p>
+
+Note that the sample uses an EVAL Apigee X Organization and hence a single Apigee X Instance only. In case you have a PROD Apigee X Organization then you will be able to easily extend the sample accordingly.
+
+You can deploy this sample by executing the shell commands listed below.  
+
+Set the GCP Project ID for the location of your Bootstrap GCP Project:
+
+```sh
+PROJECT_ID=my-bootstrap-project-id
+```
+
+Set the GCP Project ID for the location of your APIGEE GCP Project (a [Shared VPC](https://cloud.google.com/vpc/docs/shared-vpc) Service Project):
+
+```sh
+APIGEE_PROJECT_ID=my-apigee-project-id
+```
+
+Set the GCP Project ID for the location of your [Shared VPC](https://cloud.google.com/vpc/docs/shared-vpc) Host GCP Project:
+
+```sh
+HOST_PROJECT_ID=my-host-project-id
+```
+
+Draw a copy of the x-demo.tfvars to customize your Terraform Input Variables.
+
+```sh
+cp ./x-demo.tfvars ./my-config.tfvars
+cp ./infra/environments/poc/x-demo.tfvars .infra/environments/poc/my-config.tfvars
+```
+
+In case the three GCP Projects for which you set the Project IDs above do not exist yet and you do want to automate their creation you have to set the following variables in your my-config.tfvars:
+* project_create: Set this to true
+* billing_account: Set this to your billing account identifier 
+* project_parent: Set this to either 'organizations/0123456789' or 'folders/0123456789' where '0123456789' is either your GCP Organization identifier or the GCP Folder identifier respectively.
+
+Decide on a [backend](https://www.terraform.io/docs/language/settings/backends/index.html) and create the necessary config. To use a backend on Google Cloud Storage (GCS) use:
+
+```sh
+gsutil mb "gs://$PROJECT_ID-tf"
+
+cat <<EOF >terraform.tf
+terraform {
+  backend "gcs" {
+    bucket  = "$PROJECT_ID-tf"
+    prefix  = "terraform/state"
+  }
+}
+EOF
+```
+
+Validate your config:
+
+```sh
+terraform init
+terraform plan --var-file=./my-config.tfvars -var "project_id=$PROJECT_ID" -var "apigee_project_id=$APIGEE_PROJECT_ID" -var "host_project_id=$HOST_PROJECT_ID"
+```
+
+and provision the Bootstrap GCP Project:
+
+```sh
+terraform apply --var-file=./my-config.tfvars -var "project_id=$PROJECT_ID" -var "apigee_project_id=$APIGEE_PROJECT_ID" -var "host_project_id=$HOST_PROJECT_ID"
+```
+
+Create the Infrastructure as Code (takes roughly 25min):
+
+```sh
+cd ./infra/
+./setupRepo.sh $PROJECT_ID 
+```
+**Warning**: This step will kick off a Cloud Build in the Boostrap GCP Project.
+* Check the Cloud Build history in your Bootstrap GCP Project on https://console.cloud.google.com/cloud-build/builds?supportedpurview=project and make sure the build completes before continuing to the next step.
+
+Provision a simple httpbin Proxy:
+
+```sh
+cd ../app/
+./setupRepo.sh $PROJECT_ID 
```

Comment:
Maybe we need to state that the git command needs the config to be initialized?

```
git config --global user.email "you@example.com"
git config --global user.name "Your Name"
```

Reply:
Added as suggested